test/merkle_patricia_tree_test.exs

```diff
@@ -45,7 +46,7 @@ defmodule MerklePatriciaTreeTest do
     "test/support/ethereum_common_tests/TrieTests/trie#{Atom.to_string(type)}.json"
   end
 
-  def maybe_hex("0x" <> _str=hex_string), do: hex_to_binary(hex_string)
+  def maybe_hex("0x" <> hex_string), do: hex_to_binary(hex_string)
```

Comment:
Nevermind, this looks like it was right before. Though you could probably just replace the definition (now) with `Base.decode16!(case: :mixed)` and this would work.